stdlib/Makefile

```diff
@@ -41,8 +41,6 @@ DEPFLAGS=-slash
 OC_CPPFLAGS += -I$(ROOTDIR)/runtime
 
 # Object file prefix
-P=stdlib__
```

Comment:
If you remove the definition altogether, you should also remove the comment on the previous line.

Reply:
Fixed, thanks.